redfish-core/lib/oem/ibm/pcie_topology_refresh.hpp

```diff
@@ -164,42 +128,6 @@ inline void
         "PCIeTopologyRefresh", std::variant<bool>(state));
 }
 
-/**
- * @brief Retrieves Save PCIe Topology Info properties over DBUS
- *
- * @param[in] aResp     Shared pointer for completing asynchronous calls.
- *
- * @return None.
- */
-inline void
-    getSavePCIeTopologyInfo(const std::shared_ptr<bmcweb::AsyncResp>& aResp)
-{
-    crow::connections::systemBus->async_method_call(
-        [aResp](const boost::system::error_code ec,
-                std::variant<bool>& savePCIeTopologyInfo) {
-            if (ec)
-            {
-                BMCWEB_LOG_ERROR << "DBUS response error " << ec;
-                messages::internalError(aResp->res);
-                return;
-            }
-            const bool* savePCIeTopologyInfoPtr =
-                std::get_if<bool>(&savePCIeTopologyInfo);
-            if (!savePCIeTopologyInfoPtr)
-            {
-                messages::internalError(aResp->res);
-                return;
-            }
-            aResp->res.jsonValue["Oem"]["@odata.type"] =
-                "#OemComputerSystem.Oem";
-            nlohmann::json& pcieRefresh = aResp->res.jsonValue["Oem"]["IBM"];
-            pcieRefresh["@odata.type"] = "#OemComputerSystem.IBM";
-            pcieRefresh["SavePCIeTopologyInfo"] = *savePCIeTopologyInfoPtr;
-        },
-        "xyz.openbmc_project.PLDM", "/xyz/openbmc_project/pldm",
-        "org.freedesktop.DBus.Properties", "Get", "com.ibm.PLDM.PCIeTopology",
-        "SavePCIeTopologyInfo");
-}
 
```

Comment:
Fixed the build problem

Reply:
so what about PATCH code (set method)?

Reply:
PATCH is still there and will continue to be used. 

curl -k -X PATCH https://${BMC_IP}/redfish/v1/Systems/system/ -d '{"Oem":{"IBM":{"PCIeTopologyRefresh":true}}}'

It is just the GET, they might want removed since it tied /redfish/v1/Systems/system/  to PLDM